pkg/assets/config.go

```diff
@@ -0,0 +1,42 @@
+// Copyright 2022 syzkaller project authors. All rights reserved.
+// Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+package assets
+
+import "fmt"
+
+type AssetTypeConfig struct {
+	Always bool `json:"always"`
+	Never  bool `json:"never"`
+	// TODO: in future there'll also be `OnlyOn` and `NeverOn`, but so far we don't really need that.
+	// TODO: here will also go compression settings, should we ever want to make it configurable.
+}
+
+type Config struct {
+	GcsBucket string                     `json:"gcs_bucket"`
```

Comment:
This looks too GCS-specific. syz-ci can upload coverage reports to FTP. It uses a more generic "where to upload" name and them looks at the value prefix ("gs:", "ftp:") to choose where to upload.
It's better to use the same here.